src/Stryker.Core/Stryker.Core/Mutants/MutationContext.cs

```diff
@@ -148,7 +154,7 @@ public ExpressionSyntax InjectExpressionLevel(ExpressionSyntax mutatedNode, Expr
         /// <param name="originalNode">Source node, used to generate mutations</param>
         /// <returns>A mutated node containing the mutations.</returns>
         public StatementSyntax InjectBlockLevel(StatementSyntax mutatedNode, StatementSyntax originalNode)
-        => _store.PlaceBlockMutations(mutatedNode, m => originalNode.InjectMutation(m));
+            => _store.PlaceBlockMutations(mutatedNode, originalNode.InjectMutation);
 
         /// <summary>s
         /// Injects pending block level mutations for expression body method or functions
```

Comment:
Should'n be changed if now this also calls `PlaceStatementMutations` ?

Reply:
see comment above for the current naming rationale